### ISSUE TITLE
Issue with "Handling text — strings in JavaScript": prerequisites to do "Test your skills: Strings" not filled with this section

### ISSUE THREAD

**Issue by mtcarvalho** [mdn/content#3715]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings

#### What information was incorrect, unhelpful, or incomplete?

I read the section [Handling text — strings in JavaScript](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings) and tried to do the [Test your skills!](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings#test_your_skills!) section, which I finished with some difficulty because this section do not cover some topics needed to finish the exercises, like `length` and `indexOf()`. Later, I read [Useful string methods](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Useful_string_methods), which explains all the functions needed to finish the same [Test your skills!](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Useful_string_methods#test_your_skills!) section.

Thus, I think that the section to test skills should not be present in "Handling text — strings", because it confuses the reader/JavaScript student.

#### Specific section or headline?

[Handling text — strings in JavaScript](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings)
and
[Test your skills!](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings#test_your_skills!) reference

#### What did you expect to see?

I did not expect to see a link to exercises that cover functions not explained in the section.

#### Did you test this? If so, how?

N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/javascript/first_steps/strings`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/first_steps/strings/index.html
* Last commit: https://github.com/mdn/content/commit/b1ba9264fb9ce189976e9fa54af009a9b47c0480
* Document last modified: 2021-03-09T08:28:52.000Z

</details>

**Comment by chrisdavidmills** [comment]
In the "Test your skills" section on the first page you cite, it does say at the end of the section "Note that this also requires knowledge from the next article, so you might want to read that first."

We did contemplate getting rid of the "Test your skills" section altogether, but we thought it would look a bit weird to have a fundamental article will no skill tets link, so we decided to go with this instead.

I'm closing this now, but feel free to add further thoughts. Thanks!
